.ci/azure-pipelines/env.yml

```diff
@@ -99,3 +99,42 @@ jobs:
       tags: "$(tag)"
       condition: and(eq(variables['Build.Repository.Name'], 'PointCloudLibrary/pcl'),
                      eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+- job: BuildAndPushWindows
+  timeoutInMinutes: 360
+  displayName: "Env"
+  pool:
+    vmImage: 'windows-latest'
+  strategy:
+    matrix:
+      Winx86:
+        platform: x86
+        tag: winx86
```

Comment:
I think we should include 2019 in the tag because windows containers only run on compatible OS (in this case, ltsc-2019 will run on windows 2019 only)

Reply:
Is 2019 not needed since there's just 1 windows release? 😅